bin/check-mesos.rb

```diff
@@ -29,29 +29,26 @@
 require 'sensu-plugin/check/cli'
 require 'rest-client'
 
-# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
-MASTER_DEFAULT_PORT = '5050'.freeze
-SLAVE_DEFAULT_PORT = '5051'.freeze
-
 class MesosNodeStatus < Sensu::Plugin::Check::CLI
   option :server,
          description: 'Mesos servers, comma separated',
          short: '-s SERVER1,SERVER2,...',
          long: '--server SERVER1,SERVER2,...',
          default: 'localhost'
 
-  option :mode,
```

Comment:
ok , this makes it a little inconsistent but I understand. I am gonna merge this and do a major release as an option removed is a breaking change.